dbtmetabase/__init__.py

```diff
@@ -1,51 +1,104 @@
 import logging
+import sys
+import os
 
-from .dbt import DbtReader
 from .metabase import MetabaseClient
+from .parsers.dbt_folder import DbtFolderReader
+from .parsers.dbt_manifest import DbtManifestReader
 
-__version__ = "0.7.0"
+__version__ = "0.8.0"
 
 
 def export(
-    dbt_path: str,
     mb_host: str,
     mb_user: str,
     mb_password: str,
-    database: str,
-    schema: str,
-    mb_https=True,
-    sync=True,
-    sync_timeout=30,
-    includes=[],
-    excludes=[],
+    mb_database: str,
+    dbt_database: str,
+    dbt_manifest_path: str = None,
+    dbt_path: str = None,
+    schema: str = "public",
+    schemas_excludes: list = [],
+    mb_https: bool = True,
+    mb_verify: str = None,
+    sync: bool = True,
+    sync_timeout: int = None,
+    includes: list = [],
+    excludes: list = [],
+    include_tags: bool = True,
+    dbt_docs_url: str = None,
 ):
     """Exports models from dbt to Metabase.
 
     Arguments:
-        dbt_path {str} -- Path to dbt project.
         mb_host {str} -- Metabase hostname.
         mb_user {str} -- Metabase username.
         mb_password {str} -- Metabase password.
-        database {str} -- Target database name.
-        schema {str} -- Target schema name.
+        mb_database {str} -- Target mb_database name. Database in Metabase is aliased.
```

Comment:
Just an extra reminder that although we keep the `dbt_path` flow (folder) for backward compatibility, renaming this argument will be a breaking change to existing users.

Reply:
Good call. Ill emphasize the distinction between the 2 database args in the docstring and help text and revert the name to ensure seamless compatibility.